ios/Capacitor/Capacitor/Plugins/Permissions.swift

```diff
@@ -97,8 +101,11 @@ public class CAPPermissionsPlugin: CAPPlugin {
         ret = "denied"
       case .notDetermined:
         ret = "prompt"
+      #if swift(>=5.3)
+      case .ephemeral:
+        ret = "ephemeral"
       }
-      
+      #endif
```

Comment:
This `#endif` needs to be moved up. When compiled out, it's breaking the brackets on the switch statement.